src/main/java/org/fundacion/jala/converter/models/facade/ChecksumFacade.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Zacarias Huanca Alconz
+ */
+package org.fundacion.jala.converter.models.facade;
+
```

Comment:
The blank line should be deleted

Reply:
In the code convention we agreed that there has to be a blank space between the package and the imports

Reply:
This goes against our convention